admin/controllers/check-form.js

```diff
@@ -231,15 +227,15 @@ const displayCheckForm = async (req, res) => {
   }
 
   try {
-    checkWindows = await checkWindowService.getCheckWindowsAssignedToForms()
+    checkWindows = await checkWindowService.getCheckWindowsAssignedToFormsV2([formId])
```

Comment:
I agree with the added warning here.  I don't think it is good if the low level data access functions throw errors is something isn't found.  They simply do not have enough context to do that (if that is indeed what is happening)